### DISCUSSION TITLE
How to start with the first target highlighted?

### DISCUSSION THREAD

**Discussion by stesvis** [gilbarbara/react-joyride#898] [Q&A] [answered]
Hello,
I am unable to start with the first step highlighted. Instead it just shows me the red circle:
![image](https://user-images.githubusercontent.com/6062684/230447419-4d49b3fb-ee98-48ee-a71a-2611f548d985.png)

```
      <Joyride
        callback={handleJoyrideCallback}
        continuous
        run={onboardingRun}
        scrollToFirstStep
        steps={onboardingSteps}
        styles={{
          options: {
            zIndex: 10000,
          },
        }}
      />
```

The step shows the tooltip only after I click on the red circle. Any idea?

**Comment by gilbarbara** [comment, marked as answer]
Hey @stesvis 

You can use `disableBeacon` in the first step.
It's in the step [documentation](https://docs.react-joyride.com/step).
